lib/Common/ConfigFlagsList.h

```diff
@@ -966,6 +967,7 @@ FLAGNR(Boolean, CurrentSourceInfo     , "Enable IASD get current script source i
 FLAGNR(Boolean, CFGLog                , "Log CFG checks", false)
 FLAGNR(Boolean, CheckAlignment        , "Insert checks in the native code to verify 8-byte alignment of stack", false)
 FLAGNR(Boolean, CheckEmitBufferPermissions, "Check JIT code buffers at commit and decommit time to ensure no PAGE_EXECUTE_READWRITE pages.", false)
+FLAGNR(Boolean, BytecodeOptimizer, "Optimize bytecode size in the bytecode emitter", DEFAULT_CONFIG_BytecodeOptimizer)
```

Comment:
This name seems pretty generic; would any future bytecode optimizations use the same flag?